impl/proto/mutation_v1_service/gen.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:generate protoc -I=. -I=$GOPATH/src/ -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis/ --go_out=,plugins=grpc:. mutation_v1_service.proto
```

Comment:
Hmm, this seems to be neccessary on my machine to make sure that go generate uses the right version of googleapis. 
If I remove this line all calls to `grpc.Errorf` in the generated code get replaced with `status.Errorf` for instance. And I get error ala:
```
impl/proto/mutation_v1_service/mutation_v1_service.pb.gw.go:165: too many arguments in call to runtime.AnnotateContext
        have ("golang.org/x/net/context".Context, *runtime.ServeMux, *http.Request)
        want ("golang.org/x/net/context".Context, *http.Request)
``` 
while compiling. /cc @cesarghali